pkg/scheduler/scheduler.go

```diff
@@ -44,13 +42,15 @@ import (
 	internalqueue "k8s.io/kubernetes/pkg/scheduler/internal/queue"
 	"k8s.io/kubernetes/pkg/scheduler/metrics"
 	"k8s.io/kubernetes/pkg/scheduler/volumebinder"
+	utilio "k8s.io/utils/io"
 )
 
 const (
 	// BindTimeoutSeconds defines the default bind timeout
 	BindTimeoutSeconds = 100
 	// SchedulerError is the reason recorded for events when an error occurs during scheduling a pod.
-	SchedulerError = "SchedulerError"
+	SchedulerError  = "SchedulerError"
+	maxConfigLength = 10 * 1 << 20 // 10MB
```

Comment:
Just want to get check that 10MB is an appropriate value - confirming that there is no feasible reason this file would be more than 10MB?

Reply:
Although the scenario is different, I think this kind of setting is also applicable 
xref: https://github.com/kubernetes/kubernetes/pull/76519#issuecomment-483822126
cc @liggitt